validators/faq.adoc

```diff
@@ -0,0 +1,153 @@
+= Troubleshooting
+:icons: font
+:toc: macro
+
+ifdef::env-github[]
+
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+
+toc::[]
+
+endif::[]
+
+
+
+Need help solving an issue? Check to see if this has already been answered below. If you can't find an answer to your issue, reach out to the SKALE Network developer community on discord.  
+
+http://skale.chat/[Ask a question]
+
+== Validator Node
+
+=== How do I change my Node's IP address?
+
+It would be best to avoid having to change your node's IP address. It's recommended to use a permanent IP that can dynamically point to a node's machine IP (for example, elastic IP) so you can update machines if needed without having to change the IP recorded in SKALE Manager during node registration.
+
+If you must change your node IP in SKALE Manager, you must perform a node exit that could be costly to the node operator if it's currently supporting SKALE Chains.
```

Comment:
We have to mention that node exit and re-registration with the new node IP has to be completed the first 3 days of each month. And ends on the 3rd day of the month UTC midnight.